src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -231,8 +232,15 @@ impl DeviceManager {
                         return Some(device_id.to_string());
                     }
                 }
-                _ => {
-                    // TODO: support find other device type
+                DeviceType::VhostUserNetwork(device) => {
+                    if device.config.socket_path == host_path {
+                        return Some(device_id.to_string());
+                    }
+                }
+                DeviceType::HybridVsock(_) => {
+                    continue;
+                }
+                DeviceType::Vsock(_) => {
```

Comment:
I agree with that is non-relevant with this patch. But it is necessary to do that from my perspective. The reason is that there is no warning when a developer somehow forgets to give a implementation of a device type.

Reply:
Did you fix this because you got a warning? Or did you do it to make things easier for future developers?
If you insist on making the change, please put it in a separate commit. This will keep the changes clean and easy for reviewers to understand.

Reply:
Yeah, I still think it is necessary for us. Plus, I improve the code style of it. It looks more concise now.

```rust
DeviceType::HybridVsock(_) | DeviceType::Vsock(_) => {
    continue;
}
```

The changes have been laid in a separate commit now. Please take a look, thanks!